services/frontend/src/components/qualifiedPools/QualifiedPoolsView.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import PropTypes from "prop-types";
+import { Row, Col, List, Empty, Descriptions, Typography } from "antd";
+import { LinkOutlined } from "@ant-design/icons";
+
+const { Link } = Typography;
+
+const QualifiedPoolsView = ({ qualifiedPoolsInfo }) => {
+  const generateQualifiedPoolsInfoList = (dataSource) => {
+    return (
+      <List
+        itemLayout="horizontal"
+        dataSource={dataSource}
+        renderItem={(item) => (
+          <List.Item>
+            <Descriptions>
+              <Descriptions.Item>{item.classification}</Descriptions.Item>
+              <Descriptions.Item>{item.jobTitle}</Descriptions.Item>
+              <Descriptions.Item>
+                <Link href={item.jobPosterLink} target="_blank">
+                  <LinkOutlined />
+                  {item.selectionProcessNumber}
+                </Link>
+              </Descriptions.Item>
+            </Descriptions>
+          </List.Item>
+        )}
+      />
+    );
+  };
+
+  return (
+    <>
+      {qualifiedPoolsInfo.length > 0 ? (
+        <Row>
+          <Col xs={24} lg={24}>
+            {generateQualifiedPoolsInfoList(qualifiedPoolsInfo)}
+          </Col>
+        </Row>
+      ) : (
+        <Empty
+          image={Empty.PRESENTED_IMAGE_SIMPLE}
+          description={<FormattedMessage id="profile.qualified.empty" />}
+        />
+      )}
+    </>
+  );
```

Comment:
instead of abstracting this simple component with a function call, doing an if statement first could maybe make it clearer. What do you think?

```suggestion
  if (qualifiedPoolsInfo.length === 0) {
    return (
      <Empty
          image={Empty.PRESENTED_IMAGE_SIMPLE}
          description={<FormattedMessage id="profile.qualified.empty" />}
        />
    );
  }
  
  return (
    <Row>
      <Col xs={24} lg={24}>
        <List
          itemLayout="horizontal"
          dataSource={qualifiedPoolsInfo}
          renderItem={(item) => (
            <List.Item>
              <Descriptions>
                <Descriptions.Item>{item.classification}</Descriptions.Item>
                <Descriptions.Item>{item.jobTitle}</Descriptions.Item>
                <Descriptions.Item>
                  <Link href={item.jobPosterLink} target="_blank">
                    <LinkOutlined />
                    {item.selectionProcessNumber}
                  </Link>
                </Descriptions.Item>
              </Descriptions>
            </List.Item>
          )}
        />
      </Col>
    </Row>
  );
```